provider/testdata/data/postcard/fingerprints.csv

```diff
@@ -751,9 +750,8 @@ displaynames/regions@1, sr-Cyrl, 7715B, 51420e0d3503da27
 displaynames/regions@1, sr-Latn, 4776B, e5961905fd208c83
 displaynames/regions@1, th, 10811B, 84aad4d11372eb38
 displaynames/regions@1, tr, 4650B, 8f8bd5f16f02a43
-displaynames/regions@1, und, 2138B, 87850fe8d7efdd55
```

Comment:
Question: What happened to `displaynames/regions@1, und` and also `displaynames/languages@1, und`

Reply:
`und` only contained fallback mappings (`DE` -> `DE`), which are removed in 43.